src/main/java/net/minestom/server/terminal/MinestomConsoleWriter.java

```diff
@@ -0,0 +1,31 @@
+package net.minestom.server.terminal;
+
+import java.util.Map;
+
+import org.tinylog.core.LogEntry;
+import org.tinylog.writers.AbstractFormatPatternWriter;
+
+public final class MinestomConsoleWriter extends AbstractFormatPatternWriter {
```

Comment:
Again no reason to expose it, could be in `MinestomTerminal`

Reply:
> Again no reason to expose it, could be in `MinestomTerminal`

@TheMode 
It does need exposing.
It's not for `MinestomTerminal`, but for the tinylog's `Writer` that is defined in the `META-INF/services`, which is higher than `MinestomTerminal`.

Reply:
It's a hacky workaround.
I didn't find any other way to fix the `LineReader` issue.
This is the best that I could do.
Maybe one day another solid fix can be used instead.